config/test.exs

```diff
@@ -17,7 +17,7 @@ config :exq,
   redis_timeout: 5000,
   genserver_timeout: 5000,
   test_with_local_redis: true,
-  max_retries: 0,
+  max_retries: 1,
```

Comment:
Rather than changing config value for all tests, can we just change for your specific test?

Reply:
Done